java/server/src/org/openqa/selenium/docker/Docker.java

```diff
@@ -133,7 +133,7 @@ public Container create(ContainerInfo info) {
       output.write(info);
     }
 
-    LOG.info("Creating container: " + json);
+    LOG.fine("Creating container: " + json);
```

Comment:
It was transformed to `finest` in the previous PR: https://github.com/SeleniumHQ/selenium/pull/7170/files#r281602416

"Creating a container is an expensive and error-prone operation. Keeping the level at info is helpful."